src/transactions/feed/detailContent/SwapContent.tsx

```diff
@@ -0,0 +1,133 @@
+import BigNumber from 'bignumber.js'
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import TokenDisplay, { formatValueToDisplay } from 'src/components/TokenDisplay'
+import Touchable from 'src/components/Touchable'
+import OpenLinkIcon from 'src/icons/OpenLinkIcon'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { tokensListSelector } from 'src/tokens/selectors'
+import { TokenExchange } from 'src/transactions/types'
+import networkConfig from 'src/web3/networkConfig'
+
+export interface Props {
+  exchange: TokenExchange
+}
+
+// Note that this is tested from TransactionDetailsScreen.test.tsx
+export default function SwapContent({ exchange }: Props) {
+  const { t } = useTranslation()
+  const tokensList = useSelector(tokensListSelector)
+
+  const fromTokenSymbol = tokensList.find(
+    (token) => token.address === exchange.outAmount.tokenAddress
+  )?.symbol
+  const toTokenSymbol = tokensList.find((token) => token.address === exchange.inAmount.tokenAddress)
+    ?.symbol
+
+  //TODO: verifying fee part.
```

Comment:
TODO?

Reply:
I use the first element (index 0) in the `fees` array. And I think that `fees` array always has one element. But I put TODO here because I wanted to hear the others' opinions on it.